app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMisc.java

```diff
@@ -90,28 +91,25 @@ public static void addAlarm(@NonNull Context context,
             alarmTimes.add(Integer.parseInt(alarmTimeString));
         }
         long when;
-        Time time;
+        Clock clock;
         long startTime;
-        long startTimeInSeconds = lecture.dateUTC;
+      long startTimeInMilliSec = lecture.dateUTC;
 
-        if (startTimeInSeconds > 0) {
-            when = startTimeInSeconds;
-            startTime = startTimeInSeconds;
-            time = new Time();
+      if (startTimeInMilliSec > 0) {
+        when = startTimeInMilliSec;
+        startTime = startTimeInMilliSec;
+          clock = new SystemClock();
         } else {
-            time = lecture.getTime();
-            startTime = time.normalize(true);
-            when = time.normalize(true);
+            clock = lecture.getClock();
+        startTime = clock.toMilliseconds();
+        when = clock.toMilliseconds();
```

Comment:
# Indentation
Please fix the indentation by using the default formatting of Android Studio.

Reply:
Can you please move the part of the "undo" commit 9620b055d1363b611cb7206d005daf4fcf654ece into the original commit 7d28aec26cbaef326baa136546029d47531d3250 where the imports were reorganized to clean up the history?

Reply:
Resolved in "Moment" rewrite of this branch. See 29c7fd82e7a7bf2716f64ab1b1afed2f1eca7233 and following commits.